src/test/starknet.t.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// Copyright (C) 2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.6.12;
+
+import "../Goerli-DssSpell.t.base.sol";
+import "dss-interfaces/Interfaces.sol";
+
+contract ConfigStarknet {
+
+    StarknetValues starknetValues;
+
+    struct StarknetValues {
+        address core_implementation;
+        uint256 dai_bridge_isOpen;
+        uint256 dai_bridge_ceiling;
+        uint256 dai_bridge_maxDeposit;
+    }
+
+    function setValues() public {
+        starknetValues = StarknetValues({
+            core_implementation:       0x60C5fA1763cC9CB9c7c25458C6cDDFbc8F125256,
+            dai_bridge_isOpen:         1,        // 1 open, 0 closed
+            dai_bridge_ceiling:        200_000,  // Whole Dai Units
+            dai_bridge_maxDeposit:     5         // Whole Dai Units
+        });
+    }
```

Comment:
I think we should lift this to the `config.sol`

Reply:
Okay, after some discussion, I think it makes sense to break this out once it becomes large, and even it should be broken out into its own starknet config file.  Once we have MCD on these L2s, the configs should be broken out per L2.